utils/swift_build_support/swift_build_support/toolchain.py

```diff
@@ -151,13 +151,13 @@ def _release_date():
         """Return the release date for FreeBSD operating system on this host.
         If the release date cannot be ascertained, return None.
         """
-        try:
-            # For details on `sysctl`, see:
-            # http://www.freebsd.org/cgi/man.cgi?sysctl(8)
-            out = subprocess.check_output(['sysctl', '-n', 'kern.osreldate'])
-            return int(out)
-        except subprocess.CalledProcessError:
+        # For details on `sysctl`, see:
+        # http://www.freebsd.org/cgi/man.cgi?sysctl(8)
+        out = shell.capture(['sysctl', '-n', 'kern.osreldate'],
+                            dry_run=False, echo=False, optional=True)
```

Comment:
Similarly, I'd rather we stop than produce a subtly incorrect build.  The way I think about it is, either the user has specified an explicit FreeBSD version (e.g., for cross-compiling from a different OS), then this function shouldn't be even called, or we really need this information in order to proceed.